frontera/contrib/scrapy/settings_adapter.py

```diff
@@ -0,0 +1,27 @@
+from frontera.settings import BaseSettings, DefaultSettings
+from scrapy import log
+
+
+class ScrapySettingsAdapter(BaseSettings):
+    """
+    Wrapps the frontera settings, falling back to scrapy and default settings
+    """
+    def __init__(self, crawler_settings):
+        frontera_settings = crawler_settings.get('FRONTERA_SETTINGS', None)
+        if not frontera_settings:
```

Comment:
May be this check is useless now? It could be quite common after we merge that PR, that there is no ``FRONTERA_SETTINGS``.